apps/mull-api/src/app/entities/event.entity.ts

```diff
@@ -40,7 +40,7 @@ export class Event implements IEvent {
   endDate: Date;
 
   @Field()
-  @Column()
+  @Column({length: '3333' })
```

Comment:
You didn't format this file. Please run `precommit run -a` when issuing a PR.

![image](https://user-images.githubusercontent.com/23544999/104862175-946d5f80-5900-11eb-9e82-86848b5a4792.png)